data/huion-h640p.tablet

```diff
@@ -0,0 +1,36 @@
+# HUION
+# H640P
+# 
+#      *-----------------------------------*
+#    A |                                   |
+#    B |                                   |
+#    C |                                   |
+#      |              DISPLAY              |
+#    D |                                   |
+#    E |                                   |
+#    F |                                   |
+#      *-----------------------------------*
+#
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/145
+
+[Device]
+Name=Huion H640P
+ModelName=
+DeviceMatch=usb:256c:006d
+PairedIDs=usb:256c:006d
```

Comment:
you don't actually need this line here, `PairedIDs` is only required where the touch screen VID/PID isn't the same as the tablet VID/PID.